### ISSUE TITLE
official repository

### ISSUE THREAD

**Issue by darodi** [darodi/kcc#37]
This fork is merged back to the upstream repository
https://github.com/ciromattia/kcc/

I might mark this repository as archived and read-only in the future.

**Comment by xMonTersx** [comment]
@darodi How could I download the release in the future? In this repo or ciromattia repo?

**Comment by darodi** [comment]
@xMonTersx 

When github releases will be activated on the upstream directory, everything will be there.
In the meantime, all changes will be here too.

**Comment by axu2** [comment]
Yea, this should be read only so people go to the main one.

**Comment by darodi** [comment]
@axu2 
I'm still using this repository for dev tests and beta versions, so no read only yet.
The main repository is already referenced everywhere 
https://github.com/darodi/kcc/releases

**Comment by axu2** [comment]
Good point, I forgot about that. How about renaming the pinned issue to say to use main repo?
